fuse_variables/include/fuse_variables/orientation_3d_stamped.h

```diff
@@ -127,17 +139,21 @@ class Orientation3DLocalParameterization : public fuse_core::LocalParameterizati
     return true;
   }
 
+#if CERES_VERSION_AT_LEAST(2, 1, 0)
+  bool MinusJacobian(
+#else
   bool ComputeMinusJacobian(
+#endif
     const double* x,
     double* jacobian) const override
   {
     double x0 = x[0] * 2;
     double x1 = x[1] * 2;
     double x2 = x[2] * 2;
     double x3 = x[3] * 2;
-    jacobian[0] = -x1; jacobian[1]  =  x0; jacobian[2]  =  x3;  jacobian[3]  = -x2;  // NOLINT
-    jacobian[4] = -x2; jacobian[5]  = -x3; jacobian[6]  =  x0;  jacobian[7]  =  x1;  // NOLINT
-    jacobian[8] = -x3; jacobian[9]  =  x2; jacobian[10] = -x1;  jacobian[11] =  x0;  // NOLINT
+    jacobian[0] = x1; jacobian[1]  = -x0; jacobian[2]  = -x3;  jacobian[3]  =  x2;  // NOLINT
+    jacobian[4] = x2; jacobian[5]  =  x3; jacobian[6]  = -x0;  jacobian[7]  = -x1;  // NOLINT
+    jacobian[8] = x3; jacobian[9]  = -x2; jacobian[10] =  x1;  jacobian[11] = -x0;  // NOLINT
```

Comment:
This tingle my spidey senses, but I assume it makes sense to a domain expert.

Reply:
Yes, this is what I mentioned in the description that looks like a latent bug hiden by two bugs cancelling each other.

I'm testing things out to confirm that with these changes things are at least fine, although I believe they should indeed be better.

The unit tests started to fail when I used the `ceres::Manifold` implementation of the code that was "duplicated" in this repo.